src/lib/snack-bar/simple-snack-bar.scss

```diff
@@ -28,4 +28,9 @@ $mat-snack-bar-button-margin: 48px !default;
     size: inherit;
     weight: 600;
   }
+
+  /deep/ [dir='rtl'] & {
```

Comment:
I would like to avoid having `/deep/` anywhere in the library. 
(and prefer turning off ViewEncapsulation)